bitnami/postgresql-ha/templates/_helpers.tpl

```diff
@@ -524,6 +524,17 @@ Return the PostgreSQL extended configuration configmap.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the Pgpool configuration configmap.
+*/}}
+{{- define "postgresql-ha.pgpoolConfigurationCM" -}}
+{{- if .Values.pgpool.configurationCM -}}
+{{- printf "%s" (tpl .Values.pgpool.configurationCM $) -}}
```

Comment:
With this you will only allow a fixed string or something like `{{ .Values. ... }}` (but no if statements or similar). If this is expected then fine, if not then use tplValue.

Reply:
I think it's enough with values like `{{ .Values. ... }`. We're using the same approach for every custom configmap/secret.